js/components/admin-page-settings.js

```diff
@@ -64,6 +64,24 @@ const AdminPageSettings = function() {
 					metaKey="menu_position"
 				/>
 			</PanelRow>
+			<PanelRow>
+				<div className="hello-admin-select-multiple">
+					<MetaSelectControl
```

Comment:
In general, the select control kinda sucks at doing multiselect. Happy to leave as is for now, but as another option, you could try switching it out for checkboxes instead.

Checkboxes are a little harder to manage (you need to keep track of them in a state object) but I have a decent example here: https://github.com/hello-wp/hello-labs/blob/c89353c3334ec9e0c5a87e6903ef9086ef1207be/js/blocks/group-pricing-table/components/edit.js#L207-L216